package.json

```diff
@@ -58,13 +58,11 @@
     "grunt-sails-linker": "~0.10.1",
     "grunt-sync": "~0.2.3",
     "i18n": "~0.5.0",
-    "include-all": "~0.1.6",
```

Comment:
This module is not used in the core, [see](https://github.com/balderdashy/sails/search?utf8=%E2%9C%93&q=include-all). But used in [sails-build-dictionary](https://github.com/balderdashy/sails-build-dictionary/blob/master/package.json#L19)